$OpenBSD: patch-cextern_cfitsio_lib_grparser_c,v 1.1 2022/01/10 23:51:09 jca Exp $

malloc.h is not standard, even on sparc
Same for memory.h

Index: cextern/cfitsio/lib/grparser.c
--- cextern/cfitsio/lib/grparser.c.orig
+++ cextern/cfitsio/lib/grparser.c
@@ -70,11 +70,6 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#ifdef sparc
-#include <malloc.h>
-#include <memory.h>
-#endif
-
 #include <string.h>
 #include "fitsio2.h"
 #include "grparser.h"
